go.sum

```diff
@@ -1,3 +1,5 @@
+github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```

Comment:
I see this is not used anywhere - is this a leftover from local testing?

When on master branch and running a `go mod tidy && go mod vendor` it does not change anything (go 1.16.6).

Bringing this up to reduce the size of the PR a bit. Something to consider on every ticket we are working on: The reviewer has to go through all the changes - the bigger the change-set the higher the chance to miss something (important). We aim for small PRs with single responsibility.